src/tbp/monty/frameworks/models/evidence_matching.py

```diff
@@ -366,6 +366,12 @@ def reset(self):
         self.symmetry_evidence = 0
         self.last_possible_hypotheses = None
 
+        self.current_mlh["graph_id"] = "no_observations_yet"
+        self.current_mlh["location"] = [0, 0, 0]
+        self.current_mlh["rotation"] = Rotation.from_euler("xyz", [0, 0, 0])
+        self.current_mlh["scale"] = 1
+        self.current_mlh["evidence"] = 0
```

Comment:
Wha was the reason these needed to be added now?

Reply:
Since not resetting `current_mlh` was causing incorrect logging for `patch_off_object` episodes, I thought it made sense to include it here. More specifically, an episode could end up with `patch_off_object` but it would report that the most likely object was whatever the MLH was for the previous episode.

Reply:
ah okay, good catch!